garrysmod/gamemodes/sandbox/gamemode/spawnmenu/creationmenu/content/contentsidebartoolbox.lua

```diff
@@ -30,7 +30,6 @@ function PANEL:Init()
 	Button:SetSize( 20, 20 )
 	Button:SetCursor( "sizeall" )
 	Button:SetTooltip( "#spawnmenu.header_tooltip" )
-	local slot = Button:Droppable( "SandboxContentPanel" )
 

```

Comment:
```suggestion
	Button:Droppable( "SandboxContentPanel" )

```

While the variable is unused, `Button:Droppable` is actually necessary for the H1 header thing to be drag'n'droppable.